<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>volumeTool</class>
 <widget class="QWidget" name="volumeTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Tool_box" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius: 6px;   /*设置按钮的边框圆⻆为 0 像素，
实现直⻆边缘*/
background-repeat:no-repeat; /* 背景图⽚不重复平铺
*/
border: none; /*⽆边框*/
background-position:center center; /*背景图⽚放置在
按钮的中⼼位置*/
}

/*悬停状态*/
QPushButton:hover
{
background-color: rgb(40,40,40); /*设置背景颜
⾊为半透明的红⾊*/
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Volume">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-image:url(&quot;:/images/volume_on.png&quot;);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="Slider_box" native="true">
        <property name="styleSheet">
         <string notr="true">border-radius: 2px;  </string>
        </property>
        <widget class="QFrame" name="In_line">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>34</y>
           <width>180</width>
           <height>4</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="Out_line">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>34</y>
           <width>120</width>
           <height>4</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 255)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
